resources/views/partials/talk-in-list.blade.php

```diff
@@ -1,4 +1,5 @@
-<div class="pull-right">
```

Comment:
Is there any other section of the site that uses talk-in-list that expects it to not have <li> in it?